### PR TITLE
chore(ap): Bump assets parachain to `polkadot-v1.10.0`

### PR DESCRIPTION
**Pull request by al3mart**
Bump dependencies and move pallet configuration to assets module.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
-# Assets Parachain Template
+# Base Parachain Template
```
Comment (brunopgalvao) [inline README.md:1]:
```suggestion
# Assets Parachain Template
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
 
-A new [Cumulus](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus)-based Substrate node, configured with fungible and non-fungilble asset functionalities, including asset fractionalization.
+A new [Cumulus](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus)-based Substrate node.
```
Comment (brunopgalvao) [inline README.md:3]:
```suggestion
A new [Cumulus](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus)-based Substrate node, configured with fungible and non-fungilble asset functionalities, including asset fractionalization.
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
-# Assets Parachain Template
+# Base Parachain Template
 
-A new [Cumulus](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus)-based Substrate node, configured with fungible and non-fungilble asset functionalities, including asset fractionalization.
+A new [Cumulus](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus)-based Substrate node. 
 
-This project is a fork of the [Base Parachain Template](https://github.com/r0gue-io/base-parachain).
```
Comment (brunopgalvao) [inline README.md:5]: Keep this.

**Review by brunopgalvao — CHANGES_REQUESTED** [review]

**Review by brunopgalvao — APPROVED** [review]

File: network.toml
Comment (AlexD10S) [inline network.toml:1]: This shouldn't be included right? pop-cli will generate this file.

**Review by AlexD10S — APPROVED** [review]

**Review by AlexD10S — CHANGES_REQUESTED** [review]
The `network.toml `shouldn't be included right? `pop-cli` will generate this file.

**Comment by AlexD10S** [comment]
@brunopgalvao apologies for removing your approval by mistake; could you please approve it again?

**Comment by al3mart** [comment]
> The `network.toml `shouldn't be included right? `pop-cli` will generate this file.

You are right! Our contracts-parachain template needs addressing as well, then.

**Review by AlexD10S — APPROVED** [review]

**Pull request merged**